modules/geo-layers/src/mvt-layer/mvt-layer.ts

```diff
@@ -382,10 +399,10 @@ export default class MVTLayer<ExtraProps extends {} = {}> extends TileLayer<
 
   private _setWGS84PropertyForTiles(): void {
     const propName = 'dataInWGS84';
-    const tileset: Tileset2D = this.state.tileset;
+    const tileset: Tileset2D | null = this.state.tileset;
 
     // @ts-expect-error selectedTiles are always initialized when tile is being processed
-    tileset.selectedTiles.forEach((tile: Tile2DHeader & ContentWGS84Cache) => {
+    tileset?.selectedTiles.forEach((tile: Tile2DHeader & ContentWGS84Cache) => {
```

Comment:
`tileset!.selectedTiles` is better? The comment assumes that this will never be null so I would rather it crashes than silently fails if it is wrong